File: src/main/java/io/dustin/finstage/financial/dto/ExternalFinancialStatementResponse.java
package io.dustin.finstage.financial.dto;

import lombok.Getter;

import java.util.List;

@Getter
public class ExternalFinancialStatementResponse {
    private List<ExternalIncomeStatement> income_statement;
    private List<ExternalBalanceSheet> balance_sheet;
    private List<ExternalCashFlowStatement> cash_flow;
}

File: src/main/kotlin/io/dustin/finstage/financial/dto/ExternalFinancialStatementResponse.kt
import com.fasterxml.jackson.annotation.JsonProperty
import io.dustin.finstage.financial.dto.ExternalBalanceSheetResponse
import io.dustin.finstage.financial.dto.ExternalCashFlowStatementResponse
import io.dustin.finstage.financial.dto.ExternalIncomeStatementResponse

data class ExternalFinancialStatementResponse(
        @JsonProperty("income_statement")
        val incomeStatement: List<ExternalIncomeStatementResponse>,

        @JsonProperty("balance_sheet")
        val balanceSheet: List<ExternalBalanceSheetResponse>,

        @JsonProperty("cash_flow")
        val cashFlow: List<ExternalCashFlowStatementResponse>
)
